pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -966,9 +968,12 @@ func SetDefaults(cr *api.KieApp) {
 		}
 		addWebhookPwds(specApply.Objects.Servers[index].Build)
 		setJvmDefault(specApply.Objects.Servers[index].Jvm)
+		setResourcesDefaultServers(specApply.Objects.Servers[index].Resources)
 	}
 	isTrialEnv := strings.HasSuffix(string(specApply.Environment), constants.TrialEnvSuffix)
 	setPasswords(specApply, isTrialEnv)
+	setResourcesDefaultConsole(&specApply.Objects.Console)
+	setResourcesDefaultSmartRouter(specApply.Objects.SmartRouter)
```

Comment:
Wouldn't be better to keep the same format of settings the Resources defaults for all object? We keep the same format for setJvmDefault. Also there could be only one function for settting up the default Resoruces and we could give to it also the constants that's need to be set.
```suggestion
	setResourcesDefaultConsole(&specApply.Objects.Console.Resources)
	setResourcesDefaultSmartRouter(specApply.Objects.SmartRouter.Resources)
```

Reply:
ok